### DISCUSSION TITLE
Markdown format `mermaid` doesn't work on mobile app

### DISCUSSION THREAD

**Discussion by huabin** [community/community#19454] [Mobile]
### Reproduce

Add `mermaid` to a markdown file

<pre>
```mermaid
graph LR
Collect-->Organize;
Organize-->Learn;
```
</pre>

On the web, it works

![Screen Shot 2022-06-26 at 10 10 19](https://user-images.githubusercontent.com/535675/175796420-1e427ec2-e6f0-4fc6-864c-6d9072ffbcf6.png)

But on the mobile app, it looks like this

![IMG_78F4BE3173CF-1](https://user-images.githubusercontent.com/535675/175796566-16a63ac6-18d9-4864-99f4-beed6eacc697.jpeg)

### More information

GitHub mobile app version: 1.70.0(70531263)

iOS version: iOS 15 & iOS 16

**Comment by melvincwng** [comment]
@huabin, Hi Bin Hua, thanks for highlighting this to Github Community.

For bugs, please raise a ticket and kindly report it to [GitHub Bug Report](https://support.github.com/contact/bug-report).

If it's a bug, the support engineers will investigate the issue, and usually get back to you within a couple of working days.

Hope this clarifies and helps resolves your issue.

If this reply has been helpful, please 'Mark as answer' so that the rest are aware this issue has been addressed.

Thank you for your cooperation, and have a great day ahead 😄

**Reply by huabin** [reply]
Hi, thanks for the reply. 

Actually, I think this is not a bug, maybe a F.R. 

:)

**Reply by melvincwng** [reply]
@huabin, ah I see, no worries then :)

**Reply by EDM115** [reply]
#13362
